README.md

```diff
@@ -164,16 +166,22 @@ cropped along some dimension, the engine will also apply a translation according
 |`left`|If the content is wider than the view, translate it so that we see the left part.|
 |`right`|If the content is wider than the view, translate it so that we see the right part.|
 
-You can always trigger a new transformation to be applied by using the `setContentSize` or `setContainerSize` APIs.
+If, after this process, the content is smaller than the container, note that the current
+[Smaller Policy](#smaller-policy) applies.
+
+Note: you can always trigger a new transformation to be applied by using the `setContentSize` or `setContainerSize` APIs.
 
 #### Smaller policy
 
-You can also control how the content will be positioned when it is smaller than the container size through the `setSmallerPolicy(int)` method or by using the `smallerPolicy` attribute on the `ZoomLayout`.  By default, content will be centered when it is smaller than it's container.
+You can control how the content will be positioned when it is smaller than the container through 
+the `setSmallerPolicy(int)` method or by using the `smallerPolicy` XML attribute of `ZoomLayout` and `ZoomImageView`.
+By default, content is always centered when it is smaller than its container.
```

Comment:
The one thing I still find confusing when reading this documentation is **when** and **how often** this `smaller policy` applies. I thought (and the code also looks like it) that it will have an effect even after the initial base transformation is done (which is the first paragraph of the **Transformations** section) and also specifies the behavior when zooming out on a `contentView` that initially was bigger than the container.
I think this should be added to the documentation somewhere as the current text was not clearly explaining that and I again found myself digging through the code to understand how it is supposed to work.

Reply:
I think I will open a new issue later to discuss this whole smaller policy thing before next release. We will improve docs after this is sorted out